source/connectors/kafka-sink-properties.txt

```diff
@@ -0,0 +1,249 @@
+       |
+       | **Default**: ``""``
+       | **Accepted Values**: A comma-separated list of field names
+
+   * - value.projection.type
+     - string
+     - | The type of value projection to use.
+       |
+       | **Default**: ``none``
+       | **Accepted Values**: ``none``, ``blacklist``, or ``whitelist``
+
+   * - writemodel.strategy
+     - string
+     - | The class that handles how build the write models for the sink documents.
+       |
+       | **Default**: ``com.mongodb.kafka.connect.sink.writemodel.strategy.ReplaceOneDefaultStrategy``
+       | **Accepted Values**: A fully qualified Java class name
+
+   * - topic.override.<topicName>.<propertyName>
+     - string
+     - | Specifies per-topic configuration overrides. For example:
+       | ``topic.override.foo.collection=bar`` instructs the connector sink to store data from the ``foo`` topic in the ``bar`` collection. See :ref:`topic-specific-configuration` for additional examples.
+
+       .. note::
+       
+          All configuration options can be specified on a per-topic basis
+          except ``connection.uri`` and ``topics``.
+       
+       | **Default**:  ``""``
+       | **Accepted Values**: Value specific to the topic override property
+
+
+.. _topic-specific-configuration:
+
+Topic-Specific Configuration Settings
+-------------------------------------
+
+The MongoDB Kafka Sink Connector supports sinking data from multiple topics.
+Specify properties specific to a topic using configurations in the
+``topic.override.<topicName>.<propertyName>`` format. The properties that
+are not overridden will use the global (not topic-specific) settings or
+default values.
+
+.. note::
+
+   The ``topics`` property and ``connection.uri`` property are global and
+   **cannot** be overridden.
+
+The following examples demonstrate how to apply topic-specific settings to
+sink data:
+
+Example: Override Connector Sink Settings on TopicA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: properties
+
+   topic.override.topicA.collection=collectionA
+   topic.override.topicA.document.id.strategy=com.mongodb.kafka.connect.sink.processor.id.strategy.UuidStrategy
+   topic.override.topicA.post.processor.chain=com.mongodb.kafka.connect.sink.processor.DocumentIdAdder,com.mongodb.kafka.connect.sink.processor.BlacklistValueProjector
+   topic.override.topicA.value.projection.type=blacklist
+   topic.override.topicA.value.projection.list=k2,k4
+   topic.override.topicA.max.batch.size=100
+
+These properties result in the following actions for messages originating
+from the ``topicA`` Kafka topic:
+
+- Generate a UUID to be stored in the ``_id`` field for each new document.
+- Omit fields ``k2`` and ``k4`` from the value projection using a blacklist.
+- Write documents to the MongoDB collection ``collectionA`` in batches of
+  up to 100.
+
+Example: Override Connector Sink Settings on TopicC
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
These example titles could be a bit more descriptive. Why should I look at the TopicA example vs the TopicC example? It's very hard to tell without just reading through and thinking about both examples.

Reply:
Thanks, I think I will combine these examples into one that just demonstrates topic property overrides.